Src/BootCamp.Chapter/ContactsCenter.cs

```diff
@@ -19,9 +38,31 @@ public ContactsCenter(string peopleFile)
         /// <returns></returns>
         public List<Person> Filter(Predicate<Person> predicate)
         {
-            var people = new List<Person>();
-            // ToDo: implement applying filter.
-            return people;
+            return _people.Where(x => predicate(x)).ToList();
+        }
+
+        private static Person StringToPerson(string line)
+        {
+            var personData = line.Split(',');
```

Comment:
Weird formatting. Too many empty lines. Combine related blocks of code by not adding an empty line.

Reply:
Removed empty lines between personData assignment.